CHANGES.rst

```diff
@@ -25,6 +25,8 @@ Bug fixes in 23.1
   labgrid-client.
 - Fix sftp option issue in SSH driver that caused sftp to only work once per
   test run.
+- ManagedFile NFS detection heuristic now looks up canonical path on remote
+  host instead of leaving symlinks as-is.
```

Comment:
This message is easy to misunderstand: The symlink resolution happens on the *local* host. So now, the real mount paths must be identical.

Instead of "canonical representation" just call it symlink resolution where and in the commit message.